### DISCUSSION TITLE
action编译的时候，upload出问题

### DISCUSSION THREAD

**Discussion by yeenjie123456** [tw93/Pake#263] [Q&A] [answered]
# 如图。
<img width="1645" alt="CleanShot 2023-01-16 at 17 20 21@2x" src="https://user-images.githubusercontent.com/62682810/212642330-e0b1cf6c-17e4-4aa0-a369-a4f878b4cf18.png">
## 能帮忙解答下吗，急死了谢谢😂

**Comment by tw93** [comment]
参考这里
https://github.com/tw93/Pake/wiki/GitHub-Actions-%E5%9C%A8%E7%BA%BF%E7%BC%96%E8%AF%91%E5%A4%9A%E7%B3%BB%E7%BB%9F%E7%89%88%E6%9C%AC#2%E5%89%8D%E5%BE%80-actions-%E9%A1%B5%E9%9D%A2%E5%90%AF%E7%94%A8-github-actions
应该是你的权限没有开

**Reply by yeenjie123456** [reply]
># 请问您说的是这个吗😂这个开了的
> <img width="911" alt="CleanShot 2023-01-16 at 17 29 29@2x" src="https://user-images.githubusercontent.com/62682810/212644252-1856cbff-67b4-4fd4-9dc2-5b3ab9004ec5.png">

**Reply by yeenjie123456** [reply]
windows版正常上传了，就是mac和lin不行不知道为啥😂

**Reply by Tlntin** [reply]
> windows版正常上传了，就是mac和lin不行不知道为啥joy

额，这个应该是批量打包里面的名称，替换的时候把`-`识别为正则了，导致出错了，待会我修复一下。

**Reply by Tlntin** [reply]
> > windows版正常上传了，就是mac和lin不行不知道为啥joy
> 
> 额，这个应该是批量打包里面的名称，替换的时候把`-`识别为正则了，导致出错了，待会我修复一下。
已经设置好了，打包了一下没有问题，测试了linux安装也没有问题。
https://github.com/Tlntin/Pake/releases/tag/continuous

**Comment by tw93** [comment, marked as answer]
我试了下最新代码打包 1.0.2 是ok的
我看了看你的fork代码是不是你删除了一些东西 复原一下 或者重复fork试试
